content/en/storage-provider/videos/actors.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "How Filecoin Actors Work"
+description: "Overview of Filecoin Actors"
+menu:
+    storageprovider:
+        parent: "storage-provider-videos"
+---
+
+This talk given by zenground0 in July of 2022 takes a deep dive into the actors that make up the computational model of the Filecoin protocol. Learn about why Filecoin uses actors, the different types of actors, and what the current eleven available Filecoin actors do. Understand the upgrades that have occurred since mainnet launch. As the Filecoin Virtual Machine (FVM) is developed, the capabilities of Filecoin actors will greatly expand along with the potential for the Filecoin Protocol, by decoupling adding functionality from protocol upgrades and allowing for independent user actors.
+
+<iframe width="708" height="398" src="https://www.youtube.com/embed/9JbwbTPonv0" title="How Filecoin Actors Work" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [What are Filecoin actors](#what-are-filecoin-actors)
+- [The Actor Model and Blockchain Consensus](#the-actor-model-and-blockchain-consensus)
+- [Actor Code](#actor-code)
+- [Filecoin is complicated](#filecoin-is-complicated)
+- [Future Actors and Two Layers](#future-actors-and-two-layers)
+- [The FVM’s Impact](#the-fvms-impact)
+- [Verifiable Storage](#verifiable-storage)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+### What are Filecoin actors
+
+Filecoin actors are hard-coded programs that run on the Filecoin blockchain. If you understand Ethereum style smart contracts, you more or less understand Filecoin actors. Actors are written ahead of time (as of the time of writing) by Protocol Lab engineers. Actors are a key aspect of how the Filecoin blockchain protocol operates. 
+
+### The Actor Model and Blockchain Consensus
+
+Blockchain consensus is a way to figure out a shared state. Filecoin uses incremental block appends to accomplish this. You will see a bunch of blocks with some forks. These forks are not used for consensus.
+
+![actors1](https://imgur.com/6p3WEbb.png)
```

Comment:
If you're adding images, there's a couple of things you need to do:

1. Create a folder within `storage-provider/videos/` called `actors`.
2. Move this `.md` into the `actors` folder and rename it `index.md`
3. Move any images into the `actors` folder.
4. Reference the images from within the `index.md` file like `![Alt text](./example.png)`.

Reply:
Thank you for the help with this. I completed this for each video.